hw/xbox/dsp/dsp_cpu.c

```diff
@@ -1079,7 +1080,14 @@ static void dsp_write_reg(dsp_core_t* dsp, uint32_t numreg, uint32_t value)
             dsp->registers[DSP_REG_OMR] = value & 0xc7;
             break;
         case DSP_REG_SR:
-            dsp->registers[DSP_REG_SR] = value & 0xaf7f;
+
+            /* Check for unhandled bits */
+            assert((value & (1 << 7)) == 0); /* Scaling */
+            assert((value & (1 << 12)) == 0); /* Reserved */
+            assert((value & (1 << 14)) == 0); /* Double-Precision */
+            assert((value & 0xFF0000) == 0xC00000); /* EMR */
```

Comment:
nit: This should probably use constants like `DSP_SR_S` and `DSP_SR_DM` and `DSP_SR_EMR` (or individual bits from `DSP_SR_EMR`)

That way, once we implement them, we won't be able to forget to update this.